### ISSUE TITLE
Does varsignorepattern support configuring multiple parameters

### ISSUE THREAD

**Issue by lalLAL95828** [eslint/eslint#15716]
### Environment

Node version: 
npm version: 
Local ESLint version: 
Global ESLint version: 
Operating System: 

### What parser are you using?

@babel/eslint-parser

### What did you do?

```js

"rules": {
        "no-unused-vars": ["error", { "varsIgnorePattern":["^React$" ,"^child$" ] }],
    }


```

### What did you expect to happen?

I want to configure multiple varsignorepatterns

### What actually happened?

fail

### Participation

- [ ] I am willing to submit a pull request for this issue.

**Comment by aladdin-add** [comment]
does it work to use `"varsIgnorePattern": "^(React|child)$"`?

**Comment by mdjermanovic** [comment]
This works as intended, [varsIgnorePattern](https://eslint.org/docs/rules/no-unused-vars#varsignorepattern) accepts only one pattern. As noted in the previous comment, you can always merge multiple patterns into one.

> ```js
> "rules": {
>         "no-unused-vars": ["error", { "varsIgnorePattern": "^(?:React|child)$" }],
>     }
> ```
